tests/unit/mixins/I18n.ts

```diff
@@ -119,6 +135,62 @@ registerSuite('mixins/I18nMixin', {
 				});
 			}
 		},
+		'.localizeBundle() with an override': {
+			'Uses the `i18nBundle` property'(this: any) {
+				const dfd = this.async();
```

Comment:
How come we are using `this.async` here and not dealing with promises as with other async tests in widget-core? It would be super if we didn't need to use a `setTimeout` also.

Reply:
This was originally needed to workaround the fact that `localizeBundle` asynchronously loads locale-specific messages but synchronously returns either the default messages or the cached messages. However, the underlying functionality has changed between commits, so I should be able to just use `@dojo/i18n/i18n.default()` directly.